client/components/proposal/ProposalDetail.tsx

```diff
@@ -95,42 +156,84 @@ export const ProposalDetail = ({
     });
   };
 
+  const lastTx = transactions[transactions.length - 1];
+
   return (
     <>
+      <div className="sm:flex space-y-4 sm:space-y-0 justify-between items-center mb-6">
+        <div className="mr-4 relative max-w-lg">
+          <PageTitle noBottomMargin>
+            <div
+              className="break-words"
+              dangerouslySetInnerHTML={{ __html: cleanTitle }}
+            />
+          </PageTitle>
+          <div className="text-white opacity-70 text-md">
+            {id.toString().padStart(3, "0")} •{" "}
+            {`${lastTx?.event} ${moment(lastTx?.createdAt).format(
```

Comment:
If lastTx is null, this is gonna show as undefined

Reply:
Have added a render check to make sure `lastTx` is present.